Objective-C/Internal/Replicator/CBLWebSocket.mm

```diff
@@ -178,9 +178,11 @@ - (instancetype) initWithURL: (NSURL*)url
     if (self) {
         _c4socket = c4socket;
         _options = AllocedDict(options);
-        _replicator = (__bridge CBLReplicator*)context;
-        _db = _replicator.config.database;
-        _remoteURL = $castIf(CBLURLEndpoint, _replicator.config.target).url;
+        if ([(__bridge id)context isKindOfClass: [CBLReplicator class]]) {
+            _replicator = (__bridge CBLReplicator*)context;
+            _db = _replicator.config.database;
+            _remoteURL = $castIf(CBLURLEndpoint, _replicator.config.target).url;
+        }
```

Comment:
This is OK for now. A better way is https://issues.couchbase.com/browse/CBL-2945.